vissl/hooks/log_hooks.py

```diff
@@ -191,6 +191,16 @@ def on_update(self, task: "tasks.ClassyTask") -> None:
                     f"eta: {eta_string}; "
                     f"peak_mem: {peak_mem_used}M"
                 )
+                log_data = {
+                    "Rank": rank,
+                    "ep": train_phase_idx,
+                    "iter": iteration,
+                    "lr": lr_val,
+                    "loss": loss_val,
+                    "btime": batch_time,
```

Comment:
nit: `btime(ms)`

Reply:
The reason I initially renamed the keys is that if these logs are consumed by JavaScript scripts, etc. they would have to use the `log["btime(ms)"]` format instead of the simpler `log.btime` dot notation. Do you think including the units in the key (since that would be repeated in every logged frame) is better than sticking the units onto some documentation page?

Reply:
that's a very good insight @akainth015 :)

We would want to stick with the units in the key since these are quite critical aspects to how we monitor the trainings and units can often become very confusing. Hence we intentionally listed the units from the very beginning.

I see your point though. If we encounter a need in future for Javascript etc, we will revisit this :)